### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by michalxfanta**
PFW-1066

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]<Enter comprehensive title>"
+labels: bug
+assignees: ''
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: printer type: {MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2}
upgrade: {MMU2S, MMU2, MMU1}
firmware version: {3.8.1, 3.8.1 RC1, ...}

---
As a summary, please make sure you have ticked all points on this
checklist:
-->
* [ ] I searched in open and closed issues and couldn't find anything that matches.
* [ ] Printer type (MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2)
* [ ] Printer firmware version (3.8.1, 3.8.1-RC1, ...)
* [ ] Upgrade (MMU2S, MMU2, MMU1)
* [ ] Upgrade firmware version (1.0.6, 1.0.6-RC2, ...)

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: "  [ENHANCEMENT]<Enter comprehensive title>"
+labels: enhancement
+assignees: ''
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/enhancement.md:7]: printer type: {MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2}
upgrade: {MMU2S, MMU2, MMU1}

---
As a summary, please make sure you have ticked all points on this
checklist:
-->
* [ ] I searched in open and closed issues and couldn't find anything that matches.
* [ ] Printer type (MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2)
* [ ] Upgrade (MMU2S, MMU2, MMU1)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]<Enter comprehensive title>"
+labels: feature request
+assignees: ''
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]: printer type: {MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2}
upgrade: {MMU2S, MMU2, MMU1}

---
As a summary, please make sure you have ticked all points on this
checklist:
-->
* [ ] I searched in open and closed issues and couldn't find anything that matches.
* [ ] Printer type (MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2)
* [ ] Upgrade (MMU2S, MMU2, MMU1)

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: What do you want to know?
+title: "[QUESTION]<Enter comprehensive title>"
+labels: question
+assignees: ''
+
+---
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/question.md:9]: * [ ] I searched in open and closed issues and couldn't find anything that matches.

**Review by 3d-gussner — COMMENTED** [review]
Just an idea to add the check boxes

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+Please, before you create a new bug report, please make sure you searched in open and closed issues and couldn't find anything that matches.
+
+**Printer type** - [e.g. MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2]
+**Printer firmware version**-  [e.g. 3.8.1, 3.8.1-RC1, ...]
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: Add if it is an official, community or custom firmware.
Add if the issue is in specific language (translation) only.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+**MMU Upgrade** - [e.g. MMU2S, MMU2, MMU1]
+**MMU upgrade firmware version [e.g. 1.0.6, 1.0.6-RC2, ...]
+
```
Comment (3d-gussner) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: If it is a MMU specific issue maybe create issue templates also in MM-control-01 repository and add here a link to that.

**Review by DRracer — APPROVED** [review]
since @michalxfanta solved all comments with @3d-gussner , I'm approving this without changes. Further improvements may occur though.

**Pull request merged**